pythainlp/tokenize/__init__.py

```diff
@@ -139,10 +139,8 @@ def word_tokenize(
         from .pyicu import segment
 
         segments = segment(text)
-    else:  # default, use "newmm" engine
-        from .newmm import segment
-
-        segments = segment(text, custom_dict)
+    else:
+        raise ValueError()
```

Comment:
It would be good if we provide some message here. For example, 
```
f"\"{engine}\" not found.  It might be a typo; if not, please consult our document."
```